tasks/install.ps1

```diff
@@ -0,0 +1,20 @@
+[CmdletBinding()]
+Param(
+  [string]$download_url = 'https://chocolatey.org/api/v2/package/chocolatey/',
+  [boolean]$use_7zip = $False,
+  [string]$install_proxy = '',
+  [string]$seven_zip_exe = "C:\Windows\Temp\7za.exe",
+  [String]$_installdir
```

Comment:
Possibly worth noting this is a magic variable passed by bolt to access dependent files